app/javascript/angular/code/services/_sessions_utils.js

```diff
@@ -97,6 +97,15 @@ export const sessionsUtils = (
     createSessionData(session, data);
     session.loaded = true;
     callback();
+  },
+  refreshMapView: function(sessions) {
+    if (sessions.type === "MobileSessions") {
```

Comment:
wdyt about using a `case` statement?

Reply:
I don't mind either way, but I noticed it's never used in the code base and I'm not sure why. We have `else if`'s everywhere.